hs_app_netCDF/models.py

```diff
@@ -156,6 +156,19 @@ def can_have_multiple_files(cls):
         # can have only 1 file
         return False
 
+    # add resource-specific HS terms
+    def get_hs_term_dict(self):
+        # get existing hs_term_dict from base class
+        hs_term_dict = super(NetcdfResource, self).get_hs_term_dict()
+        # add new terms for NetCDF res
+        hs_term_dict["HS_NETCDF_FILE_NAME"] = ""
+        for res_file in self.files.all():
+            fn = res_file.resource_file.name
```

Comment:
@gantian127 Agreed with @pkdash. ```get_resource_file_extension(res_file)``` probably can be augmented a bit to return not only extension but also name rather than writing a new function to get name with similar logic, just a minor comment for your reference.